src/components/PanelsWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC, useEffect } from 'react';
+import Panel from '@/components/panel/Panel';
+import { useConfig } from '@/contexts/ConfigContext';
+import { getManifestUrl, getCollectionUrl } from '@/utils/panel';
+
+const PanelsWrapper: FC = ({}) => {
+  const { config, openedPanels, setOpenedPanels } = useConfig();
+
+  function initOpenedPanels(panels: Panel[]) {
+    if (setOpenedPanels) {
+      setOpenedPanels(panels);
+    }
+  }
+
+  useEffect(() => {
+    if (!config || !config.panels) {
+      console.error('Please provide the config object or the panels array in config')
+      return
+    }
+    initOpenedPanels(config.panels);
+  }, []);
+
+  
+  const panels = openedPanels ?
+    openedPanels.length > 0 &&
+    openedPanels.map((panel: Panel, i: number) => (
+      <div key={i} className="t-mr-6">
+        <Panel url={ panel.collection ? getCollectionUrl(panel): getManifestUrl(panel)} />
```

Comment:
Just pass the whole `PanelConfig` as prop here and let the component figure out what it should load. This way we can also read other upcoming config options inside it.

Reply:
Yes it is a good idea to pass the whole PanelConfig as a prop. I did it.